src/annotator/anchoring/test/text-range-test.js

```diff
@@ -273,7 +328,7 @@ describe('annotator/anchoring/text-range', () => {
         assert.equal(range.toString(), 'foobar');
       });
 
-      it('throws if start point cannot be resolved', () => {
+      it('throws if start point in same element as end point cannot be resolved', () => {
```

Comment:
There is no change to the test here. I just called out an aspect of the test setup which is important because `TextRange.toRange` has a specific code path for it. Same for the test below.